mobile/src/services/notification/notification.service.ts

```diff
@@ -0,0 +1,27 @@
+import Toast from 'react-native-toast-message';
+
+import { NotificationType } from '~/common/enums/enums';
+
+class Notification {
+  private showNotification(type: NotificationType, message: string): void {
+    Toast.show({
+      type: type,
+      text1: 'Error',
```

Comment:
Should it be for all types? Because you need more than two arguments pass them as object